sessions/js-conditions-and-booleans/booleans/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+		<title>HTML & CSS & JS</title>
+		<link rel="stylesheet" href="./css/styles.css" />
+	</head>
+	<body>
+		<h1>Exercise:</h1>
+		<p>
+			Write down what the following statements will return as a boolean. Try to figure this
+			out before putting the commands in the browser console.
+		</p>
+
+		<ul>
+			<li><code>2 == "2"</code></li>
+			<li><code>2 === "2"</code></li>
+			<li><code>2 === 2</code></li>
+			<li><code>"hello world"</code></li>
+			<li><code>""</code></li>
+			<li><code>true</code></li>
+			<li><code>0</code></li>
+			<li><code>-50</code></li>
+			<li><code>50</code></li>
+			<li><code>5 > 1</code></li>
+			<li><code>"1 > 10"</code></li>
+			<li><code>"false"</code></li>
+		</ul>
+
+		<p>To check your answers, you can run the statements with the function `Boolean()`.</p>
+	</body>
+	<script src="./js/index.js"></script>
```

Comment:
```suggestion
		<title>HTML & CSS & JS</title>
		<link rel="stylesheet" href="./css/styles.css" />
		<script src="./js/index.js" defer></script>
	</head>
	<body>
		<h1>Exercise</h1>
		<h2>
		    <code>true</code> or <code>false</code>?
		</h2>
		<p>
		   For each statement write down whether it will resolve to <code>true</code> or <code>false</code>. Once completed, check the answers in the browser console by run the statement in the <code>Boolean()</code> function. 
		</p>

		<ol>
			<li><code>2 == "2"</code></li>
			<li><code>2 === "2"</code></li>
			<li><code>2 === 2</code></li>
			<li><code>"hello world"</code></li>
			<li><code>""</code></li>
			<li><code>true</code></li>
			<li><code>0</code></li>
			<li><code>-50</code></li>
			<li><code>50</code></li>
			<li><code>5 > 1</code></li>
			<li><code>"1 > 10"</code></li>
			<li><code>"false"</code></li>
		</ol>
	</body>
```